### PR TITLE
Pass ownership to rubygems@envato.com

### PR DESCRIPTION
**Pull request by yoshdog**
This will allow other developers at envato to maintain the gem.
You should be able to push new versions of the gem using the rubygems@envato.com account.

Added contributors section on the readme to give thanks to the many
lovely contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,20 @@ details.
...
+## Contributers
+
+Many thanks to the following contributers to this gem:
+
+- Toan Nguyen - [@yoshdog](https://github.com/yoshdog)
+- Odin Dutton - [@twe4ked](https://github.com/twe4ked)
+- Sebastian von Conrad - [@vonconrad](https://github.com/vonconrad)
+- Zubin Henner- [@zubin](https://github.com/zubin)
+- Glen Tweedie - [@nocache](https://github.com/nocache)
+- Giancarlo Salamanca - [@giancarlo](https://github.com/giancarlo)
```
Comment (salamagd) [inline README.md]: Nitpick - I'm actually @salamagd on Github but I can change this myself 😉

File: README.md [hunk 2]
```diff
@@ -181,6 +181,20 @@ details.
 We welcome contribution from everyone. Read more about it in
 [`CODE_OF_CONDUCT.md`](https://github.com/envato/jwt_signed_request/blob/master/CODE_OF_CONDUCT.md)
 
+## Contributers
```
Comment (salamagd) [inline README.md]: `:%s/ontributers/ontributors/g`

**Review by salamagd — APPROVED** [review]
:+1: thanks @yoshdog!

File: README.md (hunk 1 above)
Comment (yoshdog) [reply README.md]: 😆 that may be easier

**Pull request merged**